architecture/adapters.md

```diff
@@ -68,17 +94,49 @@ The JSON API is based off of JavaScript Object Notation:
 
 ```
 
-Notice that objects are surrounded by `{curly brackets}`, and both attributes and values are surrounded by `"double quotes"`. Each attribute-value pair is specified by a single line, with a colon separating attributes and values, and attribute-value pairs are separated by a comma. Note that spacing is irrelevant to the formatting of the document except in strings -- anything between double quotes must be on a single line.
+Objects are surrounded by `{curly brackets}`, and both attributes and values are surrounded by `"double quotes"`. Each attribute-value pair is specified by a single line, with a colon separating attributes and values, and attribute-value pairs are separated by a comma. Spacing is irrelevant to the formatting of the document except in strings -- anything between double quotes must be on a single line. Finally, there is no syntax for making comments.
+
+In the `yaml-rpi` adapter, this is done through these lines in `app.rb`:
+
+```ruby
 
-In the `yaml-rpi` adapter, this is done through
+require 'sinatra/json'
+require 'oj'
+
+# ...
+	json yaml_content
+
+```
+
+`require 'sinatra/json'` and `require 'oj'` tell ruby to load the Sinatra package's JSON helper function and the Optimized JSON package respectively. Then when the adapter needs to convert the data in `yaml_content` to JSON, the `json` function from `'sinatra/json'` is called.
 
 
 ## HTTP Protocol
 
+<!-- IDK what to do here, I don't really understand this part fully yet
```

Comment:
This is just setting up the web server, and is very specific to Sinatra, the web framework we use in most of the adapters.